tomcat/src/main/tomcat-base/conf/tomcat-users.xml

```diff
@@ -1,44 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same here. Say in the PR description that you're removing tomcat-users.xml file and why.